assets/bingo/lang/pl_pl.json

```
@@ -0,0 +1,411 @@
+
+	"bingo.lobby.settings.player.tutorial.sign.line1": "Gracz",
+	"bingo.lobby.settings.player.tutorial.sign.line2": "",
+	"bingo.lobby.settings.player.tutorial.sign.line3": "Wyjaśnienie",
+	"bingo.lobby.settings.player.tutorial.sign.line4": "jak to działa",
+
+	"bingo.lobby.settings.player.tutorial.part1": "Te ustawienia umożliwiają ustawić z jakimi przedmiotami i efektami gracz zaczyna grę.\nWszystkie efekty mikstur są trwałe.",
+	"bingo.lobby.settings.player.tutorial.part2": "Istnieje już kilka gotowych konfiguracji, które możesz wykorzystać klikając na tabliczkę “Wybór/Podgląd Konfiguracji” po prawej.",
+	"bingo.lobby.settings.player.tutorial.part3": "Aby stworzyć własną konfigurację, po prostu zmień przedmioty w swoim ekwipunku lub wypij miksturę, aby dodać trwały efekt. Aby pozbyć się aktywnych efektów, użyj mleka.",
+	"bingo.lobby.settings.player.tutorial.part4": "Stań na rzeźbionym bloku kwarcu, aby zmienić tryb na kreatywny i uzyskać łatwy dostęp do wszystkich przedmiotów.\nJeśli jesteś operatorem lub grasz w trybie jednoosobowym, możesz również użyć poleceń /give i /effect.",
+	"bingo.lobby.settings.player.tutorial.part5": "Nie zapomnij zapisać swojej konfiguracji gdy skończysz - w przeciwnym razie nie zostanie ona zapisana ani wybrana.",
+
+	"bingo.lobby.settings.player.save.sign.line1": "Gracz",
+	"bingo.lobby.settings.player.save.sign.line2": "",
+	"bingo.lobby.settings.player.save.sign.line3": "Zapis",
+	"bingo.lobby.settings.player.save.sign.line4": "Konfiguracji",
+
+	"bingo.lobby.settings.player.prev": "Poprzednia",
+	"bingo.lobby.settings.player.next": "Następna",
+	"bingo.lobby.settings.player.save.title": "Zapisz Konfigurację Gracza",
+	"bingo.lobby.settings.player.save.description": "Konfiguracje do nadpisania:",
+	"bingo.lobby.settings.player.save.new": "Utwórz Nową Konfigurację",
+	"bingo.lobby.settings.player.save.new.explanation": "Ustaw nazwę dla swojej nowej konfiguracji, zmieniając nazwę otrzymanego właśnie znacznika w kowadle, które znajduje się pod tą tabliczką.\nKiedy skończysz, po prostu upuść znacznik.\nW przypadku, gdy twój ekwipunek jest pełny, możesz tymczasowo upuścić coś przed kowadłem, aby podnieść to z powrotem, gdy znacznik z nazwą będzie w GUI kowadła.",
+	"bingo.lobby.settings.player.save.new.success": "Nowa konfiguracja “%s” zapisana pomyślnie",
+	"bingo.lobby.settings.player.save.overwrite.success": "Pomyślnie nadpisano konfigurację “%s” nowymi ustawieniami.",
+
+	"bingo.lobby.settings.player.load.sign.line1": "Gracz",
+	"bingo.lobby.settings.player.load.sign.line2": "",
+	"bingo.lobby.settings.player.load.sign.line3": "Wybór/Podgląd",
+	"bingo.lobby.settings.player.load.sign.line4": "Konfiguracji",
+	"bingo.lobby.settings.player.load.title": "Załaduj Konfigurację Gracza",
+	"bingo.lobby.settings.player.load.description": "Konfiguracje do załadowania:",
+	"bingo.lobby.settings.player.load.new.success": "Konfiguracja “%s” załadowana pomyślnie",
```

```suggestion
	"bingo.lobby.settings.player.save.new.success": "Nowa konfiguracja „%s” zapisana pomyślnie",
	"bingo.lobby.settings.player.save.overwrite.success": "Pomyślnie nadpisano konfigurację „%s” nowymi ustawieniami.",

	"bingo.lobby.settings.player.load.sign.line1": "Gracz",
	"bingo.lobby.settings.player.load.sign.line2": "",
	"bingo.lobby.settings.player.load.sign.line3": "Wybór/Podgląd",
	"bingo.lobby.settings.player.load.sign.line4": "Konfiguracji",
	"bingo.lobby.settings.player.load.title": "Załaduj Konfigurację Gracza",
	"bingo.lobby.settings.player.load.description": "Konfiguracje do załadowania:",
	"bingo.lobby.settings.player.load.new.success": "Konfiguracja „%s” załadowana pomyślnie",
```
Quotation mark styling